packages/@ember/debug/index.ts

```diff
@@ -1,341 +1,12 @@
-    @return {Function} A new function that wraps the original function with a deprecation warning
-    @private
-  */
-  setDebugFunction('deprecateFunc', function deprecateFunc(...args: any[]) {
-    if (args.length === 3) {
-      let [message, options, func] = args as [string, DeprecationOptions, AnyFn];
-      return function (this: any, ...args: any[]) {
-        deprecate(message, false, options);
-        return func.apply(this, args);
-      };
-    } else {
-      let [message, func] = args;
-      return function (this: any) {
-        deprecate(message);
-        return func.apply(this, arguments);
-      };
-    }
-  });
-
-  /**
-   @module @ember/debug
-   @public
-  */
-  /**
-    Run a function meant for debugging.
-
-    Calls to this function are removed from production builds, so they can be
-    freely added for documentation and debugging purposes without worries of
-    incuring any performance penalty.
-
-    ```javascript
-    import Component from '@ember/component';
-    import { runInDebug } from '@ember/debug';
-
-    runInDebug(() => {
-      Component.reopen({
-        didInsertElement() {
-          console.log("I'm happy");
-        }
-      });
-    });
-    ```
-
-    @method runInDebug
-    @for @ember/debug
-    @static
-    @param {Function} func The function to be executed.
-    @since 1.5.0
-    @public
-  */
-  setDebugFunction('runInDebug', function runInDebug(func) {
-    func();
-  });
-
-  setDebugFunction('debugSeal', function debugSeal(obj) {
-    Object.seal(obj);
-  });
-
-  setDebugFunction('debugFreeze', function debugFreeze(obj) {
-    // re-freezing an already frozen object introduces a significant
-    // performance penalty on Chrome (tested through 59).
-    //
-    // See: https://bugs.chromium.org/p/v8/issues/detail?id=6450
-    if (!Object.isFrozen(obj)) {
-      Object.freeze(obj);
-    }
-  });
-
-  setDebugFunction('deprecate', _deprecate);
-
-  setDebugFunction('warn', _warn);
-}
-
-let _warnIfUsingStrippedFeatureFlags;
-
-if (DEBUG && !isTesting()) {
-  if (typeof window !== 'undefined' && (isFirefox || isChrome) && window.addEventListener) {
-    window.addEventListener(
-      'load',
-      () => {
-        if (
-          document.documentElement &&
-          document.documentElement.dataset &&
-          !document.documentElement.dataset['emberExtension']
-        ) {
-          let downloadURL;
-
-          if (isChrome) {
-            downloadURL =
-              'https://chrome.google.com/webstore/detail/ember-inspector/bmdblncegkenkacieihfhpjfppoconhi';
-          } else if (isFirefox) {
-            downloadURL = 'https://addons.mozilla.org/en-US/firefox/addon/ember-inspector/';
-          }
-
-          debug(`For more advanced debugging, install the Ember Inspector from ${downloadURL}`);
-        }
-      },
-      false
-    );
-  }
-}
-
+// Unfortunately export * doesn't appear to work.
```

Comment:
For posterity, can you clarify at least in the PR here in what manner `export * from` doesn't work? Otherwise one of us is going to be wondering in a year. 😂

Reply:
Looks like it does work after all. Not sure what was going on.